scripting/get5/matchconfig.sp

```diff
@@ -27,12 +27,11 @@ stock bool LoadMatchConfig(const char[] config, bool restoreBackup = false) {
     // During restore we want to keep our
     // current pauses used.
     if (!restoreBackup) {
-      g_TeamPauseTimeUsed[team] = 0;
-      g_TeamPausesUsed[team] = 0;
-      g_TeamTechPausesUsed[team] = 0;
+      g_TacticalPauseTimeUsed[team] = 0;
+      g_TacticalPausesUsed[team] = 0;
+      g_TechnicalPausesUsed[team] = 0;
     }
-    g_TechPausedTimeOverride[team] = 0;
-    g_TeamGivenTechPauseCommand[team] = false;
+    g_TechnicalPauseTimeUsed[team] = 0;
```

Comment:
Why is this outside here? Is this intentional?

Reply:
Yes, `!stop` I believe can be called during a tech timeout, so if we do a round restore, we still wish to zero out the time because we will enter a different pause state. I believe prior to refactoring there would be issues surrounding timers on restore.

Reply:
The point here is that `g_TechnicalPauseTimeUsed[team] = 0;` is always reset when a tech pause ends. It doesn't stack across multiple tech pauses, so why is it explicitly being skipped here?

https://github.com/splewis/get5/pull/784/files#diff-80fe068e671d54322735140aeae59d9c10ffea3db4bb6004365876b596355f3fR28-R35

Reply:
Ah sorry I've inverted it in my understanding. It **always** gets reset here also. Never mind. It's good.